pandas/core/generic.py

```diff
@@ -5292,46 +5295,120 @@ def replace(self, to_replace=None, value=None, inplace=False, limit=None,
                scipy < 0.18
 
         axis : {0, 1}, default 0
-            * 0: fill column-by-column
-            * 1: fill row-by-row
-        limit : int, default None.
+            Axis to interpolate along.
+
+            * 0: Fill column-by-column.
+            * 1: Fill row-by-row.
+        limit : int, default None
             Maximum number of consecutive NaNs to fill. Must be greater than 0.
+        inplace : bool, default False
+            Update the data in place if possible.
         limit_direction : {'forward', 'backward', 'both'}, default 'forward'
+            If limit is specified, consecutive NaNs will be filled in this
+            direction.
         limit_area : {'inside', 'outside'}, default None
-            * None: (default) no fill restriction
-            * 'inside' Only fill NaNs surrounded by valid values (interpolate).
-            * 'outside' Only fill NaNs outside valid values (extrapolate).
+            If limit is specified, consecutive NaNs will be filled with this
+            restriction.
+
+            * None: No fill restriction (default).
+            * 'inside': Only fill NaNs surrounded by valid values
+              (interpolate).
+            * 'outside': Only fill NaNs outside valid values (extrapolate).
+
             .. versionadded:: 0.21.0
 
-            If limit is specified, consecutive NaNs will be filled in this
-            direction.
-        inplace : bool, default False
-            Update the NDFrame in place if possible.
         downcast : optional, 'infer' or None, defaults to None
             Downcast dtypes if possible.
-        kwargs : keyword arguments to pass on to the interpolating function.
+        kwargs
+            Keyword arguments to pass on to the interpolating function.
 
         Returns
         -------
-        Series or DataFrame of same shape interpolated at the NaNs
+        Series or DataFrame
+            Same-shape object interpolated at the NaN values
 
         See Also
         --------
-        reindex, replace, fillna
+        replace : replace a value
```

Comment:
See comments above - so much can be added here

Reply:
done